cdap-docs/tracker-manual/source/index.rst

```diff
@@ -246,8 +260,8 @@ The results would appear similar to this:
   :align: center
   :class: bordered-image
 
-In this example, Tracker has found two datasets that satisfy the condition. The search
-used is shown in the upper-left, and the results show both the datasets found with
+In this example, Tracker has found a single dataset that satisfied the condition. The search
+used is shown in the upper-left, and the results show the dataset found with
 information and links for each.
```

Comment:
You have changed the text, but the image still shows two datasets returned. Plus, though you changed the text to speak of a single dataset, the ending ("links for each") is plural.

Should you go back to the original text?